### ISSUE TITLE
[Suggestion] Additional hull block type for diagonal edges

### ISSUE THREAD

**Issue by Calvert4096** [tsunamayo/Starship-EVO#2168]
I'm making an angled pylon as shown below -- the blocks currently available I think don't allow for a smooth edge here.

![capture 1](https://user-images.githubusercontent.com/65469633/82744727-47b19480-9d31-11ea-9b59-25d29394be79.png)

If we had something like the 4th hull block, but sliced in half along a diagonal plane as shown in yellow, I think that would allow for a smooth edge in this situation

![capture 2](https://user-images.githubusercontent.com/65469633/82744728-4d0edf00-9d31-11ea-9f75-a77152fe5001.png)

**Comment by tsunamayo** [comment]
Thanks for the suggestion.
It wont likely come soon, it would require two block actually, with a special code to make them work with symmetry, and also a special collision shape (they are time consuming to create atm).
Once I got this sorted out I will likely prefer to have this made via a mod on the workshop for advanced user!
Thanks

**Comment by AlienXtream** [comment]
you could make said mod yourself to serve as an "example" mod for creating advanced elements like that. much later obviously after a modloader had been implemented

**Comment by tsunamayo** [comment]
yes exactly that might be a perfect example to showcase such mod. Thanks

**Comment by Kaiser-Indrasil** [comment]
I agree - me and a few friends had a discussion on Discord the other day, highlighting numerous instances where said block was basically essential to our designs. I find it necessary in so many cases I have to otherwise use child entities for.

**Comment by Calvert4096** [comment]
For sure.  I also understand the reason for getting rid of the old-school deformable block... but damn if it wasn't useful.

I realize tsuna has bigger fish to fry right now, but it seems like this particular block should eventually be added in the base game since a surprising number of people seem to have brought it up.
